CHANGELOG

```diff
@@ -1,3 +1,7 @@
+1.1.1 Jun 19th, 2015
```

Comment:
Not mine, this was the merged changelog from upstream (goshippo/shippo-node-client). They bumped to 1.1.1, we were still on 1.1.0; I made a tag on our side called `1.1.1-shyp` to differentiate.